docs/user-guide/downward-api/index.md

```diff
@@ -88,6 +152,50 @@ This is an example of a pod that consumes its labels and annotations via the dow
 
 {% include code.html language="yaml" file="volume/dapi-volume.yaml" ghlink="/docs/user-guide/downward-api/volume/dapi-volume.yaml" %}
 
+This is another example of a pod that consumes its container's resources via the downward API volume.
+
+```yaml
```

Comment:
This seems like a clear case where it would be better to put this YAML in a separate file and use the widget to include it (so it can be directly downloaded/copied to clipboard). See line 153 of this file.